### DISCUSSION TITLE
Radeon ProRender for Blender 3.4.0

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#517] [Announcements]
- Adds support for Blender 3.1 and 3.2.
- Further options for creating fog and smoke atmosphere effects
- Options for changing shadow properties
- Hair support in RPR - Interactive

And much [More!](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/blob/master/Changelog.md#version-34)

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/releases/tag/v3.4.0'>Radeon ProRender for Blender 3.4.0</a>.</em>

**Comment by 1haris2** [comment]
please add support for noise and other texture in volume shader
